### PR TITLE
Feature/webpack

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

File: molgenis-app/package.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "molgenis-core-ui",
+  "version": "1.16.0-SNAPSHOT",
```
Comment (dennishendriksen) [inline molgenis-app/package.json]: Process resource with maven to insert version number instead of hardcoding.

File: molgenis-app/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "molgenis-core-ui",
```
Comment (dennishendriksen) [inline molgenis-app/package.json]: Should be placed in molgenis-core-ui module since the name is molgenis-core-ui?

File: molgenis-app/pom.xml [hunk 3]
```diff
@@ -60,6 +60,39 @@
...
+        		<artifactId>frontend-maven-plugin</artifactId>
+		        <version>0.0.27</version>
+		        <executions>
+			        <execution>
+                        <id>install node and npm</id>
+                        <goals>
+                            <goal>install-node-and-npm</goal>
+                        </goals>
+                        <configuration>
+                            <nodeVersion>v4.2.4</nodeVersion>
```
Comment (dennishendriksen) [inline molgenis-app/pom.xml]: Use Node.js v5?

File: molgenis-app/pom.xml [hunk 4]
```diff
@@ -60,6 +60,39 @@
...
+                            <npmVersion>2.14.15</npmVersion>
```
Comment (dennishendriksen) [inline molgenis-app/pom.xml]: Use npm v3?

File: molgenis-app/pom.xml [hunk 5]
```diff
@@ -60,6 +60,39 @@
...
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>npm install</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                    </execution>
+			      <!--	<execution>
```
Comment (dennishendriksen) [inline molgenis-app/pom.xml]: Remove comments

File: molgenis-app/pom.xml (hunk 3 above)
Comment (fdlk) [inline molgenis-app/pom.xml]: This is the LTS. Any relevant differences that we know of?

**Pull request closed without merging**